charts/weave-gitops/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
Should this ignore our source doc folder as well?

Reply:
I am not sure what you mean by the source doc. This was just auto filled.

Reply:
The source code folder.  `src`

Reply:
Oh wait nvm, I just noticed that this is under a sub directory.  Disregard.